test/unit/basic-tests.js

```diff
@@ -756,7 +756,7 @@ describe('writers', function () {
           buffers.push(buf);
           totalLength += buf.length;
           if (cb) {
-            setTimeout(cb, 50);
+            setTimeout(cb, 20);
```

Comment:
Is there a reason to change to 20? I think a code comment would be good to be more clear about this.

Reply:
I saw a failure on a CI run related to the final timeout on this test and the solution was to use `helper.setIntervalUntil()` to avoid race conditions.
I've reduced the time on this timeout to make it faster, any timeout (not `setImmediate()`) would work.
I could add a comment :)